robolectric/src/main/java/org/robolectric/plugins/DefaultSdkPicker.java

```diff
@@ -116,8 +116,9 @@ protected Set<Sdk> configuredSdks(Config config, UsesSdk usesSdk) {
         throw new IllegalArgumentException(
             "Package targetSdkVersion=" + appTargetSdk + " < minSdkVersion=" + appMinSdk);
       } else if (appMaxSdk != 0 && appTargetSdk > appMaxSdk) {
-        throw new IllegalArgumentException(
+        System.err.println(
```

Comment:
Maybe use `org.robolectric.util.Logger.warn`?
```suggestion
        Logger.warn(
```

Reply:
I tried to be consistent with the `SdkCollection` which is already used by the `DefaultSdkPicker` and uses `System.err` to print a warning when an SDK is not supported. Maybe I can use a similar message like `"[Robolectric] WARN: API level %d is not available. Tests will be run on SDK %d instead"`?

Reply:
If `System.err` is already used in these classes, I think it's fine to keep it like that for the moment. We can align later in a dedicated PR if needed.
For the error message itself, for me both options are fine